environment.dev.yml

```diff
@@ -4,19 +4,10 @@ channels:
     - conda-forge
 dependencies:
     - bump2version=1.0.1
-    - flake8=4.0.1
     - ipykernel=6.9.0
     - jupyter=1.0.0
-    - mypy=0.941
-    - numpydoc=1.1.0
-    - numpy=1.22.3
-    - pandas=1.3.5
-    - pytest=6.2.5
-    - pytest-cov=3.0.0
-    - python=3.10
-    - scikit-learn
```

Comment:
why sklearn and numpy are removed ?

Reply:
and pytest ?

Reply:
Because of the dependance of mapie installed with `pip install -e`, we also import these packages.